src/ceph-daemon

```diff
@@ -1576,31 +1576,32 @@ try:
 except NameError:
     av = sys.argv[1:]
 
-args = parser.parse_args(av)
+if __name__ != 'nosetests':
```

Comment:
:-1: for this quirk.

Reply:
yeah i don't like it either. ultimately this should be weighed against the pain of shipping around and/or a user running a modularized ceph-daemon.